src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -112,6 +120,19 @@ def __init__(self, streams, stage, area, overlap, settings_obs=None, log_path=No
             self._fn_bs, self._fn_ext = udataio.splitext(filename)
             self._log_dir = os.path.dirname(self._log_path)
 
+        if zsteps:
+            try:
+                self._focus = model.getComponent(role='focus')
```

Comment:
That's fine, but another way, possibly a little bit more generic would be to use the stream `.focuser` (if it's not `None`). If the focuser is None, then the stream is acquired only once, without using this parameter. It make it easy to indicate which stream should have this effect and which stream not (for instance, we probably want to avoid this for SEMStreams).